src/formatDate.js

```diff
@@ -50,7 +50,74 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [,,, oldSeparator] = fromFormat;
+  const [,,, newSeparator] = toFormat;
```

Comment:
```suggestion
  const oldSeparator = fromFormat.pop();
  const newSeparator = toFormat.pop();
```
it's easier that way, isn't it?

Reply:
This mutates the original array. Maybe then it's better to use const oldSeparator = fromFormat[4] ?

Reply:
Ok)